addons/l10n_in/models/account_invoice.py

```diff
@@ -98,3 +102,13 @@ def _l10n_in_get_warehouse_address(self):
         # TO OVERRIDE
         self.ensure_one()
         return False
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_l10n_in_except_once_post(self):
+        # Prevent deleting entries once it's posted for Indian Company only
+        if any(m.country_code == 'IN' and m.posted_before for m in self):
+            if self._context.get('force_delete'):
+                moves_name = ", ".join("%s (%s)" % (m.name, m.id) for m in self)
+                _logger.info(_('Force Delete Journal Entrys %s by %s (%s)', moves_name, self.env.user.name, self.env.user.id))
```

Comment:
```suggestion
                _logger.info(_('Force deleted Journal Entries %s by %s (%s)', moves_name, self.env.user.name, self.env.user.id))
```
We could inform about amount and partner as well?

Reply:
Yes, it's useful let add it